app/src/main/java/com/github/epfl/meili/messages/LatestMessageActivity.kt

```diff
@@ -13,22 +14,22 @@ class LatestMessagesActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_latest_messages)
 
-        verifyUserIsLoggedIn()
+       verifyUserIsLoggedIn()
     }
 
     private fun verifyUserIsLoggedIn() {
```

Comment:
If this function isn't useful (since it's commented out), we should remove it.

Reply:
If I do cirrus complains that this function is too similar to the mainActivity onCreate() and that I should refactor them together. What solution do you suggest?

Reply:
Isn't it a duplicate of line 37 to 40 of the same file ? In this case u can make a function with these three lines and call it twice.